executor/runtime/types/container.go

```diff
@@ -219,6 +223,16 @@ func NewTitusInfoContainer(taskID string, titusInfo *titus.ContainerInfo, resour
 		c.extraUserContainers, c.extraPlatformContainers = NewExtraContainersFromPod(*c.pod)
 	}
 
+	pConfig := podCommon.Config{}
+	c.podConfig = &pConfig
```

Comment:
Any reason to break this into two separate lines?

Reply:
nope